hibernate-core/src/main/java/org/hibernate/engine/jdbc/internal/StatementPreparerImpl.java

```diff
@@ -199,11 +205,7 @@ private void setStatementTimeout(PreparedStatement preparedStatement) throws SQL
 	}
 
 	private JdbcServices getJdbcService() {
```

Comment:
do we still need the `getJdbcService()` method ?

Reply:
probably not let me check

Reply:
I'll remove it and add another little change